### PR TITLE
Lon 1314

### PR DESCRIPTION
**Pull request by vmartos**
#### Things to review:
- [ ] Unit tests
- [ ] No hardcoded resources (strings, sizes, colors...)
- [ ] Sorted usings
- [ ] Add summaries where needed 

If needed, you can add new checkboxes to https://github.com/fluendo/VAS/blob/master/.github/PULL_REQUEST_TEMPLATE.md

### CODE REVIEW

File: VAS.UI.Gtk2/UI/Component/LicenseBannerView.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+		}
+
+		void HandleLicenseTextSizeAllocated (object o, Gtk.SizeAllocatedArgs args)
+		{
+			if (lastSize != args.Allocation.Width) {
+				lastSize = args.Allocation.Width;
+				int size = Math.Max (args.Allocation.Width - TEXT_OFFSET, TEXT_MIN_SIZE);
+				size = Math.Min (size, TEXT_MAX_SIZE);
+				licenseTextLabel.WidthRequest = size;
+				licenseTextLabel.WidthChars = size - CHARSWIDTH_OFFSET;
```
Comment (vguzmanp) [inline VAS.UI.Gtk2/UI/Component/LicenseBannerView.cs]: Is this copypasted from the other place that did this? If so, move it to an extension method for the Gtk.Label

File: VAS.UI.Gtk2/gtk-gui/gui.stetic [hunk 2]
```diff
@@ -3066,4 +3112,55 @@
...
+    <property name="Visible">False</property>
+    <child>
+      <widget class="Gtk.HBox" id="hbox1">
+        <property name="MemberName" />
+        <property name="Spacing">20</property>
+        <child>
+          <widget class="Gtk.Alignment" id="alignment1">
+            <property name="MemberName" />
+            <child>
+              <placeholder />
```
Comment (vguzmanp) [inline VAS.UI.Gtk2/gtk-gui/gui.stetic]: Why is this empty?

File: VAS.UI.Gtk2/gtk-gui/VAS.UI.Component.LicenseBannerView.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+			this.licenseTextLabel.Justify = ((global::Gtk.Justification)(2));
+			this.hbox1.Add (this.licenseTextLabel);
+			global::Gtk.Box.BoxChild w2 = ((global::Gtk.Box.BoxChild)(this.hbox1 [this.licenseTextLabel]));
+			w2.Position = 1;
+			// Container child hbox1.Gtk.Box+BoxChild
+			this.upgradeButton = new global::Gtk.Button ();
+			this.upgradeButton.CanFocus = true;
+			this.upgradeButton.Name = "upgradeButton";
+			this.upgradeButton.UseUnderline = true;
+			this.upgradeButton.Label = global::VAS.Core.Catalog.GetString ("Upgrade now");
```
Comment (vguzmanp) [inline VAS.UI.Gtk2/gtk-gui/VAS.UI.Component.LicenseBannerView.cs]: As you overwrite this, probably it shouldn't have a value, or at least shouldn't be translatable

File: VAS.Core/Interfaces/Service/ILicenseCustomizationService.cs [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+	/// <summary>
+	/// License customization service interface. use to retrieve objects based on current license status
+	/// </summary>
+	public interface ILicenseCustomizationService
+	{
+		/// <summary>
+		/// Gets the logo name.
+		/// </summary>
+		/// <returns>The logo name.</returns>
+		string GetLogoName ();
```
[the remark this answers is not in the document]
Comment (vmartos) [reply VAS.Core/Interfaces/Service/ILicenseCustomizationService.cs]: It returns just the name: "lm-basic" -pro -starter -any

File: VAS.Core/Interfaces/Service/ILicenseCustomizationService.cs [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+		/// <summary>
+		/// Gets the license information text.
+		/// </summary>
+		/// <returns>The license information text.</returns>
+		string GetLicenseInformationText ();
```
[the remark this answers is not in the document]
Comment (vmartos) [reply VAS.Core/Interfaces/Service/ILicenseCustomizationService.cs]: I thought services should only provide methods, i can change it to properties and have the logic in the getter

File: VAS.UI.Gtk2/gtk-gui/gui.stetic (hunk 2 above)
Comment (vmartos) [reply VAS.UI.Gtk2/gtk-gui/gui.stetic]: it is just an alignment to center the license information text

File: VAS.Core/Interfaces/Service/ILicenseCustomizationService.cs (hunk 5 above)
[the remark this answers is not in the document]
Comment (vguzmanp) [reply VAS.Core/Interfaces/Service/ILicenseCustomizationService.cs]: Good point, I'm not sure really.

File: VAS.UI.Helpers.Gtk2/ExtensionMethods.cs [hunk 6]
```diff
@@ -83,6 +86,30 @@ public static void Center (this Gtk.Dialog dialog)
...
 		}
 
+		/// <summary>
+		/// Set Label to change its size dynamically, by specifiying a minimum width and a maximum width, if maximum is not
+		/// specified then it will get the first allocated width of the label.
+		/// </summary>
+		/// <param name="label">Label.</param>
+		/// <param name="minWidth">Minimum width.</param>
+		/// <param name="maxWidth">Max width.</param>
+		public static void DynamicSize (this Label label, int minWidth, int maxWidth = -1)
```
Comment (vguzmanp) [inline VAS.UI.Helpers.Gtk2/ExtensionMethods.cs]: Nice!

File: VAS.Core/Resources/Styles/Sizes.cs [hunk 7]
```diff
@@ -33,6 +33,8 @@ public class Sizes
 		public const int WelcomeIconsFirstRow = 3;
 		public const int WelcomeTextHeight = 20;
 		public const int WelcomeMinWidthBorder = 30;
+		public const int WelcomeBarIconSize = 55;
```
Comment (vguzmanp) [inline VAS.Core/Resources/Styles/Sizes.cs]: Is this validated with jluna? As far as I can see in the design, the preferences icon should be 40x40

**Pull request merged**